background/setBackground.js

```diff
@@ -1,5 +1,38 @@
-const setBackground = (weatherCode) => {
-	const background = document.getElementById("background");
-	background.style.backgroundImage = `url(${backgroundsMapping[weatherCode]})`;
-	background.style.backgroundColor = '#efefef';
+const background = document.getElementById("background");
+
+const toggleBackground = (weatherCode) => {
+    if (weatherCode) {
+        setWeatherBackground(weatherCode);
+    }  else {
+        setInitialBackground();
+    }
+}
+
+const setWeatherBackground = (weatherCode) => {
+    const initialBackground = document.getElementById('initial-background');
+    if (document.body.contains(initialBackground)) {
+        document.body.removeChild(initialBackground);
+    }
+    background.style.display = 'flex';
+    background.style.backgroundImage = `url(${backgroundsMapping[weatherCode]})`;
+}
+
+const setInitialBackground = () => {
+    const newObject = document.createElement("object");
+    const finalObject = Object.assign(newObject, {
+        id: "world-map-background",
+        type:"image/svg+xml",
+        data: "./assets/Initial_background/worldMap.svg",
+        alt: "SVG world map.",
+    });
+    
+    const divContainer = document.getElementById("initial-background");
+    divContainer.innerHTML = `
+    <div id="headline">
+        <span>Check the weather around the World.</span>
+    </div>
+    `;
+    divContainer.appendChild(finalObject);
+    
+    background.style.display = 'none';
```

Comment:
please modify classes instead of adding styles

Reply:
In this function I'll use:
`background.classList.add('hide-content');`
and where I want to show the content:
`background.classList.remove('hide-content');`
Will it be ok?